evalml/tests/automl_tests/test_automl.py

```diff
@@ -160,6 +160,7 @@ def test_search_results(X_y_regression, X_y_binary, X_y_multi, automl_type, obje
                 np.dtype("float64"),
                 np.dtype("bool"),
                 np.dtype("O"),
```

Comment:
do any of these tests check if "search_order" has the correct values?

Reply:
@jeremyliweishih Line 109 checks: ``assert automl.results["search_order"] == [0, 1]`` :)